docs/docsite/rst/dev_guidelines.rst

```diff
@@ -299,6 +299,7 @@ that should be included into almost all AWS modules:
 
```

Comment:
Github won't let me put this comment on a line you didn't change, but two lines above this "There are two" should probably be updated to say "There are four".